protonvpn_cli/utils.py

```diff
@@ -24,7 +24,7 @@
 def call_api(endpoint, json_format=True, handle_errors=True):
     """Call to the ProtonVPN API."""
 
-    api_domain = "https://api.protonvpn.ch"
+    api_domain = get_config_value("USER", "api_domain").rstrip("/")
```

Comment:
What other sections are there? is `USER` the most appropriate?

Reply:
"metadata"

It should've been "Settings" and "metadata", but it's too late now :P